Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -173,6 +177,106 @@ def get_asset_details_command(client: Client, args: dict) -> CommandResults:
     )
 
 
+def reformat_readable_output(script_res: list) -> None:
+    """
+    Reformat the human_readable output so that each command appears as a separate row in the table.
+
+    Args:
+        script_res (list): The result from the polling command.
+    """
+    reformated_results = []
+    for response in script_res:
+        results = response.outputs.get('results')
+        for res in results:
+            new_d = {}
+            for key in EXECUTE_COMMAND_READABLE_FIELDS:
+                new_d[key] = res.get(key)
+            new_d['command'] = new_d['command'][1:]
+            reformated_results.append(new_d)
+    script_res[0].readable_output = tableToMarkdown(f'Script Execution Results - {script_res[0].outputs["action_id"]}',
+                                                    reformated_results, EXECUTE_COMMAND_READABLE_FIELDS, removeNull=True,
+                                                    headerTransform=string_to_table_header)
+
+
+def new_executed_command(result: dict) -> dict:
```

Comment:
I think a better naming would be - 'core_execute_command_reformate_command_data()'